invenio_previewer/extensions/video_videojs.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2023 Northwestern University.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Render video file using videojs."""
+
+from flask import current_app, render_template
+
+from ..proxies import current_previewer
+from ..utils import dotted_exts
+
+# The video file world is complex, because support depends on
+# hardware, OS, software and browser. A reasonable choice of supported
+# extensions (https://en.wikipedia.org/wiki/HTML5_video#Browser_support) that
+# are least likely to lead to an unsupported warning for end-users are
+# mp4 and webm files. However, this is made customizable per instance.
+previewable_extensions = current_app.config.get("PREVIEWER_VIDEO_EXTS", ["mp4", "webm"])
```

Comment:
The hardcoded lists of supported file extensions might limit flexibility, how about considering making it more configurable so it can be overridable on the instance level, WDYT?

Reply:
Same as above :wink:

Reply:
Same as above :)